annotate.el

```diff
@@ -69,9 +72,9 @@ See https://github.com/bastibe/annotate.el/ for documentation."
 
 (define-key annotate-mode-map (kbd "C-c C-s") 'annotate-show-annotation-summary)
 
-(define-key annotate-mode-map (kbd "C-c ]") 'annotate-next-annotation)
+(define-key annotate-mode-map (kbd "C-c ]") 'annotate-move-next-annotation)
```

Comment:
I think "move" is ambiguous, as it might be interpreted as moving the annotation. Perhaps "goto" is better?
```suggestion
(define-key annotate-mode-map (kbd "C-c ]") 'annotate-goto-next-annotation)
```

Reply:
Hello!

I agree it is better to rename, this is a case where 'goto' is not considered harmful! ;-D